src/CoreFx.Private.TestUtilities/src/CoreFx.Private.TestUtilities.csproj

```diff
@@ -29,6 +29,9 @@
     <Compile Include="$(CommonPath)\System\PasteArguments.cs">
       <Link>Common\System\PasteArguments.cs</Link>
     </Compile>
+    <Compile Condition="'$(TargetGroup)' == 'uap'" Include="$(CommonTestPath)\System\Threading\ThreadTestHelpers.cs">
```

Comment:
>clude="$(CommonTestPath)\System\Threading\ThreadTestHelpers.cs [](start = 53, length = 62)

is it possible to move this file under the sources of this library CoreFx.Private.TestUtilities? and delete it from common. this may require touching some other test projects that using this file.